instance/migrations/0047_generic_logging3.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django_extensions.db.fields
+import instance.models.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contenttypes', '0002_remove_content_type_name'),
+        ('instance', '0046_generic_logging2'),
+    ]
+
+    def data_forward(apps, schema_editor):
+        """
+        Copy log entries from GeneralLogEntry, InstanceLogEntry, and ServerLogEntry into the new
+        combined LogEntry table.
+        """
+
+    def data_backward(apps, schema_editor):
+        """
+        Reverse the data migration
+        """
```

Comment:
@bradenmacdonald I think you can remove `data_forward` and `data_backward` from this migration.